BuildResidentialHPXML/measure.rb

```diff
@@ -5767,7 +5767,7 @@ def self.set_heat_pumps(hpxml_bldg, args)
       end
     end
 
-    if [HPXML::HVACTypeHeatPumpAirToAir, HPXML::HVACTypeHeatPumpMiniSplit].include? heat_pump_type
+    if [HPXML::HVACTypeHeatPumpAirToAir, HPXML::HVACTypeHeatPumpMiniSplit, HPXML::HVACTypeHeatPumpGroundToAir].include? heat_pump_type
       compressor_type = args[:heat_pump_cooling_compressor_type]
```

Comment:
Need to update the argument description for `heat_pump_cooling_compressor_type` to include ground-to-air systems.